pkg/utils/experimentHelpers.go

```diff
@@ -17,18 +17,28 @@ func (expDetails *ExperimentDetails) SetValueFromChaosExperiment(clients ClientS
 }
 
 //SetENV sets ENV values in experimentDetails struct.
-func (expDetails *ExperimentDetails) SetENV(engineDetails EngineDetails, clients ClientSets) {
+func (expDetails *ExperimentDetails) SetENV(engineDetails *EngineDetails, clients ClientSets) error {
 	// Get the Default ENV's from ChaosExperiment
 	log.Infoln("Getting the ENV Variables")
-	expDetails.SetDefaultEnv(clients)
+	if err := expDetails.SetDefaultEnv(clients); err != nil {
+		return err
+	}
 	// OverWriting the Defaults Varibles from the ChaosEngine ENV
-	expDetails.SetEnvFromEngine(engineDetails.Name, clients)
+	if err := expDetails.SetEnvFromEngine(engineDetails.Name, clients); err != nil {
```

Comment:
smart coding.. :( but I would still prefer: 

```go
err := func
if err != nil {
  return err
} 
```

Reply:
this is for reducing the number of line, BCH issue